amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/validation/PayloadsInApiCollector.scala

```diff
@@ -60,6 +60,8 @@ class PayloadsInApiCollector(model: BaseUnit) {
     }.toSeq
   }
 
+  private def isStrictAnyShape(shape: AnyShape) = shape.meta == AnyShapeModel
```

Comment:
This method should include the fields check, otherwise the name should be "isAnyShape" or something like that

Reply:
why fields check? The only way to know if an AnyShape is in fact an AnyShape is to check the instance's metamodel.